bcl/inc/bc_font_common.h

```diff
@@ -10,18 +10,29 @@ typedef struct
     const uint8_t *image;
     uint8_t width;
     uint8_t heigth;
-    /*uint8_t dont_know;*/
 } tImage;
 
- typedef struct  {
-     uint16_t code;
-     const tImage *image;
-     } tChar;
+typedef struct  {
+    uint16_t code;
+    const tImage *image;
+} tChar;
 
- typedef struct  {
-     uint16_t length;
-     const tChar *chars;
-     } tFont;
+typedef struct  {
+    uint16_t length;
+    const tChar *chars;
+} tFont;
+
+//
+// Put another generated fonts here
+//
+extern const tFont bc_font_ubuntu8;
+extern const tFont bc_font_ubuntu12;
+extern const tFont bc_font_ubuntu14;
+extern const tFont bc_font_ubuntu24;
+
+// Defines for backward compatibility
+#define FontBig bc_font_ubuntu24
```

Comment:
Every name in BigClown SDK starts with name of the module and all defines are in uppercase . So it should be BC_FONT_BIG ... btw it is not appropriate word for font size - I rather suggest to use BC_FONT_SIZE_LARGE